src/Microsoft.FeatureManagement/VariantAllocation/Allocation.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement.VariantAllocation
```

Comment:
Is that namespace intentional?

Reply:
I mentioned this to Ross [above](https://github.com/microsoft/FeatureManagement-Dotnet/pull/250#discussion_r1290516010) but I wasn't sure what a better name would be, or if we should make a whole Variants folder like he mentioned.

Reply:
Removed it